### DISCUSSION TITLE
Sync between devices

### DISCUSSION THREAD

**Discussion by stdedos** [ImranR98/Obtainium#1638] [Q&A]
Hello there!

I know it's a bit of a weird/complicated question - but is possible / are the plans to somehow sync the "tracked apps" list between devices?

e.g. I have NewPipe on both my phones, and it'd be convenient to only have to add it once (also the GH api key, etc etc)

**Comment by ImranR98** [comment]
No plans for this currently because the install status of the apps are stored in the same place as all other app info. We would have to separate them if we wanted sync (since install status would differ between devices). Even if this is ever implemented, it would just be to some filesystem location - the actual sync between devices would be up to the user to deal with (probably via Syncthing).

**Reply by stdedos** [reply]
Even that would be amazing 😍 

Ofc, some care for secrets would be nice (probably user controls the whole stack, but still I'd be nice to have some kind of protection)
